Map debug renderers stuff

IotaBread opened the pull request:
I plan on doing more of it tomorrow

IotaBread committed:
Map some things in VillageDebugRenderer

IotaBread committed:
Map constants in BeeDebugRenderer

IotaBread committed:
PathfindingDebugRenderer mappings

IotaBread committed:
StructureDebugRenderer mappings

IotaBread committed:
GameEventDebugRenderer mappings

IotaBread committed:
ChunkLoadingDebugRenderer mappings

IotaBread committed:
GoalSelectorDebugRenderer and HeightmapDebugRenderer mappings

IotaBread committed:
GameTest, RaidCenter, VillageSections and SkyLight DebugRenderers mappings

ByMartrixx commented:
Done, I don't think there is anything left that I can map

liach commented on mappings/net/minecraft/client/render/debug/GameEventDebugRenderer.mapping:
```diff
@@ -12,8 +12,18 @@ CLASS net/minecraft/class_5739 net/minecraft/client/render/debug/GameEventDebugR
 	METHOD method_33088 addListener (Lnet/minecraft/class_5716;I)V
 		ARG 1 positionSource
 		ARG 2 range
+	METHOD method_33089 drawBoxIfCameraReady (Lnet/minecraft/class_238;FFFF)V
```
What is camera ready? Like this is not hidden by the frustum?

IotaBread replied:
I named it like that because the actual code is inside an `if (camera.isReady())`

liach commented on mappings/net/minecraft/client/render/debug/StructureDebugRenderer.mapping:
```diff
@@ -1,4 +1,11 @@
 CLASS net/minecraft/class_870 net/minecraft/client/render/debug/StructureDebugRenderer
+	FIELD field_32916 RANGE I
 	FIELD field_4624 client Lnet/minecraft/class_310;
+	FIELD field_4626 structureBoundingBoxesByDimension Ljava/util/Map;
+	FIELD field_4627 structurePiecesBoundingBoxesByDimension Ljava/util/Map;
```
```suggestion
	FIELD field_4626 structureBoundingBoxes Ljava/util/Map;
	FIELD field_4627 structurePiecesBoundingBoxes Ljava/util/Map;
```
the `byDimension` suffix is extraneous and we don't really need to emphasize it's a per-dimension lookup if this is just the general store of bounding boxes, etc.

liach commented on mappings/net/minecraft/village/VillagerGossips.mapping:
```diff
@@ -27,6 +27,9 @@ CLASS net/minecraft/class_4136 net/minecraft/village/VillagerGossips
 		ARG 2 gossipTypeFilter
 	METHOD method_19074 entries ()Ljava/util/stream/Stream;
 	METHOD method_20651 decay ()V
+	METHOD method_35120 getEntityReputationsAssociatedGossips ()Ljava/util/Map;
```
```suggestion
	METHOD method_35120 getEntityReputationAssociatedGossips ()Ljava/util/Map;
```

IotaBread committed:
Apply suggestions from code review

Co-authored-by: liach <7806504+liach@users.noreply.github.com>

IotaBread committed:
getNamesOfVillagersWithPotentialJobSite -> getNamesOfJobSitePotentialOwners and
getNamesOfVillagersWithPointOfInterest -> getNamesOfPointOfInterestTicketHolders

IotaBread committed:
Merge branch '1.17.1-pre1' into 2/1.17.1-pre1
